src/octopeer-github/main/MainController.ts

```diff
@@ -146,20 +149,22 @@ class MainController implements OptionsObserver {
     /**
      * Posts a message to the current database. If it's a HTMLPageEvent message, truncate its content before logging it in debug mode.
      * @param message the message to be posted
+     * @param prUrl the pull request URL that will be used for the DatabaseAdapter
      */
-    private postToDatabase(message: EventObject) {
+    private postToDatabase(message: EventObject, prUrl: string) {
         const success = function() {
             if (message.type === "HTMLPageEvent") {
-                Logger.debug(`Successfully logged HTML page to database: \n${(<HTMLPageEvent>message.data).dom.substring(0,1500)}...`);
+                Logger.debug(`Successfully logged HTML page to database (${prUrl}): \n
+                    ${(<HTMLPageEvent>message.data).dom.substring(0,1500)}...`);
```

Comment:
Is 1500 the DB limit? are we certain that this will log all pages, including future ones?